### PR TITLE
[MWPW-136526] MAX pages enhancement

### PR DESCRIPTION
**Pull request by qiyundai**
✅ List block: Enable dynamic pricing values by locale in the FAQ block (call to pricing sheet)
✅ Pricing summary block (feature variant): add column in offer sheet to trigger superscript next to current price (not base price)
✅ Pricing summary block (feature variant): enable suppression of border around Premium card
✅ Mobile marquee block: expand CTA to full width
✅ Marquee, FAQ and Banner blocks: enable dynamic phone number values based on locale (country); for locales (countries) where we don't have a sales phone number, fall back to US number. (pending sheet)
✅ Columns block: Align each element horizontally.

Resolves: [MWPW-136526](https://jira.corp.adobe.com/browse/MWPW-136526)

Test URLs:
- Before: https://stage--express--adobecom.hlx.page/drafts/qiyundai/business-draft?lighthouse=on
- After: https://mwpw-136526--express--adobecom.hlx.page/drafts/qiyundai/business-draft?lighthouse=on

### CODE REVIEW

File: express/blocks/columns/columns.js [hunk 1]
```diff
@@ -263,6 +263,65 @@ export default function decorate($block) {
...
+          if (entry.isIntersecting) {
+            if (columns?.length > 0) {
+              columns.forEach((col) => {
+                const childDivs = col.querySelectorAll(':scope > *');
+                if (childDivs?.length > 0) {
+                  childDivs.forEach((div) => {
+                    const referrer = div.className || div.tagName;
+                    const targetEl = referrer === 'PICTURE' ? div.querySelector('img') : div;
+                    elementsMinHeight[referrer] = Math.max(
+                      elementsMinHeight[referrer],
```
Comment (JingleH) [inline express/blocks/columns/columns.js:285]: same as the one below and the pricing-summary.js, I personally feel like this nesting is too much and would do some early returns to reduce nesting. But it's a probably preference thing

**Review by JingleH — APPROVED** [review]

File: express/blocks/columns/columns.js (hunk 1 above)
Comment (vhargrave) [reply express/blocks/columns/columns.js:285]: agree, this looks like pyramid code :)

File: express/blocks/columns/columns.js [hunk 2]
```diff
@@ -263,6 +263,65 @@ export default function decorate($block) {
...
+        'columns-iconlist': 0,
+      };
+
+      const onIntersect = (entries, observer) => {
+        entries.forEach((entry) => {
+          if (entry.isIntersecting) {
+            if (columns?.length > 0) {
+              columns.forEach((col) => {
+                const childDivs = col.querySelectorAll(':scope > *');
+                if (childDivs?.length > 0) {
```
Comment (vhargrave) [inline express/blocks/columns/columns.js:280]: nit: `if (childDivs.length)`

File: express/scripts/utils/pricing.js [hunk 3]
```diff
@@ -105,6 +106,26 @@ function getCountry() {
...
+
+  const numbersMap = await fetch('/express/system/business-sales-numbers.json').then((r) => r.json());
+
+  if (!numbersMap?.data) return;
+
+  tags.forEach((a) => {
+    const r = numbersMap.data.find((d) => d.country === getCountry());
+
+    if (!r) {
+      a.textContent = a.href.replace('tel:', '').trim();
```
Comment (vhargrave) [inline express/scripts/utils/pricing.js:120]: did we ask for any type of fallback number?

**Review by apganapa — APPROVED** [review]

**Pull request merged**